src/arcsf/data/data_module.py

```diff
@@ -0,0 +1,207 @@
+import torch
+from torch.utils.data import ConcatDataset, Dataset
+
+from arcsf.data.tofu import load_tofu
+
+_dataset_dict = {"tofu": load_tofu}
+
+
+def get_data(
+    dataset_name,
+    granularity,
+    stratified,
+    forget_random,
+    forgotten_author_fraction,
+    forgotten_fact_fraction,
+    random_seed,
+):
+    load_func = _dataset_dict[dataset_name]
+    data = load_func(
+        granularity,
+        stratified,
+        forget_random,
+        forgotten_author_fraction,
+        forgotten_fact_fraction,
+        random_seed=random_seed,
+    )
+
+    return data
+
+
+def QAformatter_basic(QA: tuple[str]) -> str:
+    """
+    Basic QA formatter which accepts a tuple outputs:
+
+    "Question: [input question]\nAnswer: [input answer]"
+
+    args:
+        - QA: Tuple of question answer pair
+    returns:
+        - full_text: formatted question--answer pair
+    """
+    Question, Answer = QA
+    question_formatting = "Question: " + Question
+    answer_formatting = "\nAnswer: " + Answer
+    full_text = question_formatting + answer_formatting
+    return full_text
```

Comment:
Could be a lot shorter using an f-string:

```suggestion
    return f"Question: {question}\nAnswer: {answer}"
```

Reply:
done